jujugui/static/gui/src/app/components/added-services-list-item/added-services-list-item.js

```diff
@@ -96,7 +96,11 @@ YUI.add('added-services-list-item', function() {
           <img src={service.icon} className="inspector-view__item-icon" />
           <span className="inspector-view__item-count">
             {service.unit_count}
-          </span> {service.name}
+          </span>
+          {' '}
```

Comment:
I guess I could have, although a natural space will always match the font spacing, as opposed to a padding where we'd have to fudge it.